app/controllers/feeds_controller.rb

```diff
@@ -89,7 +89,7 @@ def set_feed
 
   # Only allow a list of trusted parameters through.
   def feed_params
-    params.fetch(:feed, {}).permit(
+    nilify params.fetch(:feed, {}).permit(
```

Comment:
I wrote this helper awhile back, to make sure we store "nil" in the database when you leave a field blank in the UI.  Added to all forms.